src/components/formSections/FormTerms.js

```diff
@@ -56,9 +63,17 @@ class FormTerms extends React.Component {
     return null;
   }
 
+  handleError = (error) => {
+    const { stripes: { store } } = this.props;
+    // stopSubmit reports the error to redux-form and sets invalid flag to true which helps us in disabling the submit button
+    if (error) {
+      store.dispatch(stopSubmit('EditLicense', { 'customProperties': error }));
```

Comment:
There's an argument to made to taking a second (or even third) argument to this function so that TermsListField can call it with the form and field name it gets passed to it in `input.name` and `meta.form`. Up to you, but it does mean you get to avoid the magic string of `EditLicense`.